src/main/java/org/zalando/planb/revocation/api/ExceptionsResource.java

```diff
@@ -25,6 +26,14 @@
 @Slf4j
 public class ExceptionsResource {
 
+    @ExceptionHandler(RevocationUnauthorizedException.class)
+    @ResponseStatus(HttpStatus.UNAUTHORIZED)
```

Comment:
For consistency, this should be FORBIDDEN as in GLOBAL revocations (see [here](https://github.com/zalando/planb-revocation/blob/master/src/main/java/org/zalando/planb/revocation/api/impl/RevocationResourceImpl.java#L150) and [here](https://github.com/zalando/planb-revocation/blob/master/src/main/java/org/zalando/planb/revocation/api/ExceptionsResource.java#L104)).